doc/tutorials/hires_howto.rst

```diff
@@ -0,0 +1,543 @@
+.. include:: ../include/links.rst
+
+.. _hires_howto:
+
+================
+Keck-HIRES HOWTO
+================
+
+
+Overview
+========
+
+This tutorial goes through a full run of PypeIt on one of our example Keck/HIRES datasets.
+Specifically, we shows the reduction of the ``J0100+2802_H204Hr_RED_C1_ECH_-0.82_XD_1.62_1x2``
+dataset, which are observations of the quasar J0100+2802 at z=6.29 taken with the HIRES RED
+cross-disperser, echelle angle of -0.82°, cross-disperser angle of 1.62°, and 1x2
+(spectral x spatial) binning.  See :ref:`here <dev-suite>` to find this example dataset.
+
+If you're having trouble reducing your data, we encourage you to try going through this tutorial
+using this example dataset first.  Please join our `PypeIt Users Slack <pypeit-users.slack.com>`__
+(using `this invitation link
+<https://join.slack.com/t/pypeit-users/shared_invite/zt-1kc4rxhsj-vKU1JnUA~8PZE~tPlu~aTg>`__)
+to ask for help, and/or `Submit an issue`_ to Github if you find a bug!
+
+The following was performed on a Macbook Pro with 16 GB RAM. The main reduction took
+approximately 1 hour and 30 minutes, and the fluxing took an additional 20 minutes.
+
+Setup
+=====
+
+Organize data
+-------------
+
+Identify the folder where the raw data are stored and make sure you have all the calibration files
+you need, in addition to the science ones. In this example, the raw data are stored in the folder
+``PypeIt-development-suite/RAW_DATA/keck_hires/J0100+2802_H204Hr_RED_C1_ECH_-0.82_XD_1.62_1x2``.
+
+.. note::
+
+    This folder can include data from different datasets (e.g., observations taken with various
+    echelle angles, cross-disperser angles, etc). The script :ref:`pypeit_setup` (see next step)
+    will help to parse the desired dataset.
+
+Run ``pypeit_setup``
+--------------------
+
+The first script to run with PypeIt is :ref:`pypeit_setup`, which examines the raw files and generates
```

Comment:
this reminds me we need a HOWTO on the `setup` GUI

Reply:
That's a good point!